.golangci.yml

```diff
@@ -150,6 +150,9 @@ issues:
     - text: "Expect WriteFile permissions to be 0600 or less"
       linters:
         - gosec
+    - text: 'shadow: declaration of "(err|ctx)" shadows declaration at'
```

Comment:
The hardest part for me is that my IDE does not have the nuance to not care about `err` shadow. I will have to get used to the green colour. Otherwise, godbolt shows it has no impacts 👍 
Second generalised PSA is that `ineffectual assignment to err (ineffassign)`/golang compiler will prevent any other bad usage of shadow err